src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx

```diff
@@ -270,175 +195,4 @@ describe('<FilesPage>', () => {
     );
     expect(tree.props.children).to.be.null;
   });
-
-  it('should hide requested files when closed', () => {
-    const claim = {
```

Comment:
I think we want this test we just need to update the claim for lighthouse

Reply:
Done